one fake build failure, but also one real one
```
ERROR: test_notify_event (corehq.apps.app_manager.tests.test_notifications.NotificationsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_notifications.py", line 20, in test_notify_event
    notify_event('domain', couch_user, 'app_id', 'unique_form_id', message)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/notifications.py", line 25, in notify_event
    message = u'{} (<a href="{}" target="_blank">{}</a>)'.format(message, doc_url, what)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)
```

--- a/corehq/apps/app_manager/tests/test_notifications.py
+++ b/corehq/apps/app_manager/tests/test_notifications.py
@@ -19,8 +19,8 @@ class NotificationsTests(SimpleTestCase):
             message = u'Émilie, vous avez de nouveaux messages.'
             notify_event('domain', couch_user, 'app_id', 'unique_form_id', message)
 
-            notification = ('Émilie, vous avez de nouveaux messages. (<a href="https://confluence.dimagi.com/'
-                            'display/internal/App+Builder+Notifications" target="_blank">what is this?</a>)')
+            notification = (u'Émilie, vous avez de nouveaux messages. (<a href="https://confluence.dimagi.com/'
+                            u'display/internal/App+Builder+Notifications" target="_blank">what is this?</a>)')
             json_patch.dumps.assert_called_with({
                 'domain': 'domain',
                 'user_id': '123',

--- a/corehq/apps/app_manager/views/notifications.py
+++ b/corehq/apps/app_manager/views/notifications.py
@@ -19,7 +19,6 @@ def notify_form_changed(domain, couch_user, app_id, unique_form_id):
 
 
 def notify_event(domain, couch_user, app_id, unique_form_id, message):
-    message = message.encode('utf-8') if isinstance(message, unicode) else message
     doc_url = 'https://confluence.dimagi.com/display/internal/App+Builder+Notifications'
     what = _('what is this?')
     message = u'{} (<a href="{}" target="_blank">{}</a>)'.format(message, doc_url, what)

